src/cmd/ksh93/tests/comvar.sh

```diff
@@ -689,10 +689,10 @@ c.c=()
 exp=$'(\n\ttypeset -C -a c\n)'
 [[ $(print -v c) == "$exp" ]] || log_error 'setting compound array c.c=() does not preserve -C attribute'
 
-compound xx=(this=that integer x=5)
-exp=$'{\n\t"this": "that",\n\t"x": 5\n}'
-[[ $(print -j xx) == "$exp" ]] || log_error "got $(print -j xx)" expected "$exp"
-
-compound y=(foo=bar;compound x=(lef=one right=2);integer z=5)
-exp=$'{\n\t"foo": "bar",\n\t"x": {\n\t\t"lef": "one",\n\t\t"right": "2"\n\t},\n\t"z": 5\n}'
-[[ $(print -j y) == "$exp" ]] || log_error "got $(print -j y)" expected "$exp"
+# compound xx=(this=that integer x=5)
```

Comment:
Probably should have a "TODO" comment explaining why this is disabled and linking to a relevant issue.